coderkevin/fresh-data
# Add redux to package.json

This adds redux and react-redux as package dependencies.

To Test:
1. `npm install`.
2. Ensure `node_modules/redux` exists.
3. Ensure `node_modules/react-redux` exists.
